tests/e2e/opamp/run.sh

```
@@ -0,0 +1,454 @@
+REPO_ROOT="$(cd "$SCRIPT_DIR/../../.." && pwd)"
+WORK_DIR=""
+SERVER_PID=""
+FF_PID=""
+PASS_COUNT=0
+FAIL_COUNT=0
+
+# --- Configuration ---
+FF_BINARY="${FF_BINARY:-$REPO_ROOT/target/release/ff}"
+SERVER_PORT=14180
+SERVER_ADDR="http://127.0.0.1:${SERVER_PORT}"
+OPAMP_ENDPOINT="${SERVER_ADDR}/v1/opamp"
```

Shellcheck (0.11.0)

[warning] OPAMP_ENDPOINT appears unused. Verify use (or export if used externally).

(SC2034)